app/javascript/components/Study.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+const descriptionCharLimit = 750
+
+export function formatDescription(rawDescription) {
+
+  let textDescription = stripTags(rawDescription)
+  let suffixString = ''
+  if (textDescription.length > descriptionCharLimit) {
+    suffixString = '... (continued)'
+  }
+  return `${textDescription.substring(0, descriptionCharLimit)}${suffixString}`
+}
+
+function stripTags(rawString) {
+  let tempDiv = document.createElement('div');
+  // Set the HTML content with the providen
+  tempDiv.innerHTML = rawString;
+  // Retrieve the text property of the element
+  return tempDiv.textContent || '';
+}
+
+export default function Study(props) {
+  const studyDescription = { __html: formatDescription(props.study.description) }
+
+  return (
+    <div key={props.study.accession}>
+      <label htmlFor={props.study.name} id= 'result-title'>
+        <a href={props.study.study_url}>{props.study.name} </a></label>
+      <div>
+        <span id='cell-count' className='badge badge-secondary'>{props.study.cell_count} Cells </span>
+      </div>
+      <span dangerouslySetInnerHTML={studyDescription} id='descrition-text-area' disabled accession = {props.study.name}></span>
```

Comment:
Naive question: why use `dangerouslySetInnerHTML` instead of putting content between tags, i.e.:

```suggestion
      <span id='description-text-area' disabled accession = {props.study.name}>{studyDescription}</span>
```

Also note `descrition` -> `description`.

Reply:
The returned value for studyDescription has inner html tags. Taken from [ReactJS](https://reactjs.org/docs/dom-elements.html),  "dangerouslySetInnerHTML is React’s replacement for using innerHTML in the browser DOM"

Reply:
Helpful reference, thank you!

Given that, it seems the new `stripTags` function makes `dangerouslySetInnerHTML` unnecessary.

Reply:
great point